cmd/webhook/main.go

```diff
@@ -88,7 +94,17 @@ func NewDefaultingAdmissionController(ctx context.Context, _ configmap.Watcher)
 	)
 }
 
-func NewValidationAdmissionController(ctx context.Context, _ configmap.Watcher) *controller.Impl {
+func NewValidationAdmissionController(ctx context.Context, cmw configmap.Watcher) *controller.Impl {
+	// Decorate contexts with the current state of the config.
+	authStore := authorization.NewStore(logging.FromContext(ctx).Named("config-authoriztion-store"))
+	authStore.WatchConfigs(cmw)
+
+	// A function that infuses the context passed to Validate/SetDefaults with custom metadata.
+	ctxFunc := func(ctx context.Context) context.Context {
+		// return v1.WithUpgradeViaDefaulting(store.ToContext(ctx))
```

Comment:
nit: remove commented line?

Reply:
Done.

I had left it because it was in the original.